telemetry/definitions/commonDefinitions.json

```diff
@@ -2687,6 +2687,11 @@
                 }
             ]
         },
+        {
+            "name": "aws_extensionUninstalled",
```

Comment:
`aws_extension` is a new prefix. I think we could use `toolkit_uninstall` ("toolkit" is understood to mean "extension")